lib/vtls/schannel.c

```diff
@@ -83,9 +83,25 @@ static Curl_recv schannel_recv;
 static Curl_send schannel_send;
 
 #ifdef _WIN32_WCE
-static CURLcode verify_certificate(struct connectdata *conn, int sockindex);
+/* certificate validation on CE doesn't seem to work right; we'll
+ * do it following a more manual process. */
+static const int DEFAULT_MANUAL_CRED_VALIDATION = 1;
```

Comment:
Please use a #define instead of a static const int variable that is not really needed in my opinion.